assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -65,9 +66,18 @@ const ColumnContainer = styled.div`
   display: grid;
   grid-gap: 10px;
   grid-template-columns: 1fr 1fr;
-  overflow: auto;
+  overflow-x: auto;
+  overflow-y: scroll;
   padding: 0 1.5em 0 1.5em;
   margin-top: 1em;
+  &:hover {
```

Comment:
Based on the [article](https://www.filamentgroup.com/lab/scrollbars/) that is shared in the description of the issue #1698, `-webkit-overflow-scrolling: touch;` should work for auto-hiding the scrollbar, however, when testing it out, it didn't really work and came out as invalid property. Looks like this is actually [only supported in Safari](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling) and thus not really useful.

If I understand the UX intent correctly then the scrollbars should actually never always show.
Tested out an option that should work for this:
- For elements that can have scrolling, always display `overflow-y: scroll`
- By default make the scrollbar transparent (to still leave room for the scrollbar and not shift the content when the scrollbar appears)
- On hover, display the scrollbar color.

![scroll-test](https://user-images.githubusercontent.com/3294597/87312799-56dae480-c521-11ea-9f3a-04b74d3068b1.gif)

@barklund @merapi Let me know if you think this could be a good option to address this issue or if you know of a better alternative.

Reply:
FWIW, that article also says that some JS might be needed; which I tested in #1858. If it's useful, feel free to take over / copy.

Reply:
Hmm, yes, I didn't test with JS (saw that part though), I tested assuming that obtrusive scrollbars are on and then assuming they're off, however, the `-webkit-overflow-scrolling: touch;` which should be responsible for auto-hiding the scrollbar seems to be not supported.

What was the reason for not using #1858 btw? Could that still be used -- in case you got it to work with elsewhere than Safari as well?

Is it confirmed that it should respect the OS settings? I know that the issue description states that, however, based on Sam's comments https://github.com/google/web-stories-wp/issues/1698#issuecomment-652774232 and https://github.com/google/web-stories-wp/issues/1698#issuecomment-652769356 it seems as if it should always auto-hide. Can't find this in Figma though to confirm. Thoughts?

Reply:
> What was the reason for not using #1858 btw? Could that still be used -- in case you got it to work elsewhere than Safari as well?

Lack of time for testing / pursuing further. IIRC preliminary testing showed that the JS technique works, but styling the scrollbars based on that didn't seem to fully work. Just needs to be re-tested.

> Can't find this in Figma though to confirm. Thoughts?

I recommend asking Paul & Sam on the issue to get confirmation so we're 100% sure on the direction.

Seems like both are right: we shouldn't show scrollbars if it can be avoided, but respecting the OS setting is important for accessibility.